salt/modules/rpm.py

```diff
@@ -478,16 +478,22 @@ def info(*packages):
                 continue
             key, value = line
             key = key.replace(' ', '_').lower()
+            if key != 'name' and filter_attrs and key not in filter_attrs:
```

Comment:
At the moment nothing is missing for generic user and nothing for SUSE Manager. If somebody will need this, patch is welcome, of course.